common.py

```diff
@@ -85,7 +85,6 @@ def handle_file(original, cropped_folder, moved_orignals_folder, del_uncrop, FWH
                     os.rename(original, moved_original_file)
 
 def fwhm_from_filename(filename):
-    return float(5)
```

Comment:
This is an issue that is corrected here.
I did not notice this hardcoded value.
Perhaps I will write unittests once this matures, so it can be prevented.